completions/lvm

```diff
@@ -895,7 +895,7 @@ _comp_cmd_lvm()
                 vgdisplay | vgexport | vgextend | vgimport | vgmerge | vgmknodes | vgreduce | \
                 vgremove | vgrename | vgs | vgscan | vgsplit | lvchange | lvcreate | lvdisplay | \
                 lvextend | lvreduce | lvremove | lvrename | lvresize | lvscan)
-                "_${words[1]}"
+                "_comp_cmd_${words[1]}" "${words[@]:1}"
```

Comment:
I believe we need to pass `"${words[1]}" "${@:2}"` as arguments. The completion function is supposed to receive the command name (`$1`), the current word (`$2`), and the previous word if any (`$3`). Also, we need to shift the content of `COMP_WORDS`, etc.

Actually, we already have a utility to process all of these handlings. I guess we can call that utility:

```suggestion
                _comp_command_offset 1
```

Reply:
Thank you for the suggestion! That looks much better and works great too